packages/contracts-por/package.json

```diff
@@ -5,13 +5,20 @@
   "private": true,
   "scripts": {
     "clean": "rm -rf ./build && hardhat clean",
+    "lint": "yarn lint:sol && yarn lint:ts",
+    "typecheck": "tsc --noEmit",
+    "lint:sol": "solhint 'contracts/**/*.sol' && prettylint 'contracts/**/*.sol'",
+    "lint:js": "eslint '{test,scripts}/**/*.js'",
```

Comment:
We only code in TypeScript, so linting JavaScript is redundant.

Reply:
Done.